dexbot/node_manager.py

```diff
@@ -0,0 +1,52 @@
+from websocket import create_connection as wss_create
+from time import time
+
+import multiprocessing as mp
+import pandas as pd
+
+max_timeout = 2.0 # max ping time is set to 2
+
+def wss_test(node):
+    """
+    Test websocket connection to a node
+    """
+    try:
+        start = time()
+        rpc = wss_create(node, timeout=max_timeout)
+        latency = (time() - start)
+        return latency
+    except Exception as e:
+        # print(e) # suppress errors
+        return None
+
+
+def check_node(node):
+    """
+    check latency of an individual node
+    """
+    print('#', end='', flush=True)
```

Comment:
`print()` should be replaced with logger calls

Reply:
replaced with logging